Sources/KituraCompression/Compression.swift

```diff
@@ -113,8 +113,8 @@ public class Compression : RouterMiddleware {
     
     private func compress(_ inputData: Data, method: String) -> Data? {
         
-        return inputData.withUnsafeBytes() { (bytes: UnsafePointer<Bytef>) -> Data? in
-            let mutableBytes = UnsafeMutableRawPointer(mutating: bytes).assumingMemoryBound(to: Bytef.self)
+        return inputData.withUnsafeBytes() { (bytes: UnsafeRawBufferPointer) -> Data? in
+            let mutableBytes = UnsafeMutableRawPointer(mutating: bytes.baseAddress!).assumingMemoryBound(to: Bytef.self)
```

Comment:
I know baseAddress shouldn't return nil but I still think it would be better to unwrap it in a guard instead of risking crashing an application:
```
guard let baseAddress = bytes.baseAddress else { return nil }
let mutableBytes = UnsafeMutableRawPointer(mutating: baseAddress).assumingMemoryBound(to: Bytef.self)
```

Reply:
Thanks, updated.